src/components/flash/flash-legacy.spec.js

```diff
@@ -1,46 +1,66 @@
 import React from 'react';
 import TestUtils from 'react-dom/test-utils';
-import Flash from './flash';
 import { shallow, mount } from 'enzyme';
-import { elementsTagTest, rootTagTest } from '../../utils/helpers/tags/tags-specs';
-import Portal from './../portal';
-import guid from '../../utils/helpers/guid';
+import FlashLegacy from './flash-legacy.component';
```

Comment:
you could just import `Flash` and leave everything as it is. It's default export, so the name does not matters.

Reply:
I used FlashLegacy name to marked that this is Legacy component